Update removing.adoc (#8406)
owncloud/client: docs/modules/ROOT/pages/removing.adoc

= Removing the Desktop Synchronization Client

== Removing the Configuration File

> When you remove the client - the configuration file remains on your system. If you then decide to install the client again, you won't have to enter the connection information.

Reviewer: ```suggestion
When you remove the client - the configuration file remains on your system. If you then decide to install the client again, you won't need to re-enter the connection information.
```

> In case you want a clean removal of the client - delete the configuration file located at: 

Reviewer: ```suggestion
In case you want a clean removal of the client - manually delete the configuration file located at: 
```

> ----

Reviewer: ```suggestion
```

> %appdata%\owncloud\owncloud.cfg

Reviewer 2: Would recommend to just remove the whole folder.

Author: good point

Reviewer 3: The config dir is platform dependent. The location is documented somewhere in here.

Author: I think we should document it for the windows users primarily. Or for now.

Reviewer 3: https://doc.owncloud.com/desktop/advanced_usage/configuration_file.html

> %appdata%\owncloud\owncloud.cfg

Reviewer: ```suggestion
```

> ----

Reviewer: ```suggestion
```
